pam_lastlog2/src/Makemodule.am

```diff
@@ -1,5 +1,5 @@
-securelibdir = $(libdir)/security
-securelib_LTLIBRARIES = pam_lastlog2.la
+securelibexecdir = $(libdir)/security
+securelibexec_LTLIBRARIES = pam_lastlog2.la
 
```

Comment:
It would be better to move the directory definition to ./configure.ac (and meson.build) and use
`usrsecurelib_execdir` as name (for compatibility with other paths).

My suggestion (autotools only):
```diff
diff --git a/configure.ac b/configure.ac
index 23594ff23..ed285d83c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -120,6 +120,10 @@ AS_CASE([$libdir],
 )
 AC_SUBST([usrlib_execdir])
 
+usrsecurelib_execdir='${usrlib_execdir}/security'
+AC_SUBST([usrsecurelib_execdir])
+
+
 # static configuration maintaned by packages (e.g. /usr/lib)
 AC_ARG_VAR([SYSCONFSTATICDIR],
           [Path to static system configuration, default ${prefix}/lib])
diff --git a/pam_lastlog2/src/Makemodule.am b/pam_lastlog2/src/Makemodule.am
index f9a9b6b41..57fce35c1 100644
--- a/pam_lastlog2/src/Makemodule.am
+++ b/pam_lastlog2/src/Makemodule.am
@@ -1,5 +1,4 @@
-securelibdir = $(libdir)/security
-securelib_LTLIBRARIES = pam_lastlog2.la
+usrsecurelib_execdir_LTLIBRARIES = pam_lastlog2.la
 
 pam_lastlog2_la_SOURCES = \
        pam_lastlog2/src/pam_lastlog2.c

```

Reply:
Ack. But shouldn't an empty `usrsecurelib_execdir_LTLIBRARIES` also go into `Makemodule.am` and then in `pam_lastlog2` we can use `+=`.

Reply:
Yes, it's probably better (more robust) to define the generic variables as empty in the top-level Makefile.am and everywhere else, use `+=.`

I planned to release v2.40.2 today, but it seems we need to fix this and the rename issue before the release :-)

Reply:
Done.

> I planned to release v2.40.2 today, but it seems we need to fix this and the rename issue before the release :-)

Indeed.

Do you intend to also include the libuuid fixes in v2.40.2?

Reply:
Hmm ... the latest libuuid fixes are missing the branch. We need it, too, in 2.40.2 (at least conditional uuid_time64; duplicate liuuid_la_LDFLAGS). Would you like me to prepare the PR (or will I do it tomorrow)?

Reply:
I think the clear-cache-on-fork change is also important.
If you could do the PR, that would be great.